index.html

```diff
@@ -20,25 +26,40 @@
     </header>
 
     <section>
-        <img src="Assets/Ellipse 11.png" class="deep_left">
-        <img src="Assets/Rectangle 4.png" class="light_right">
-        <img src="Assets/Ellipse 3.png" class="deep_right">
-        <img src="Assets/Rectangle 6.png" class="light_left">
-      <h1 class="intro">I'm Max <br> Glad to see you</h1>
-      
+      <img src="Assets/Ellipse8.png" class="small_bal" alt="small_ball_1"/>
+      <img src="Assets/Ellipse14.png" class="small_ball" alt="small_ball_2"/>
+      <img src="Assets/Ellipse5.png" class="ball" alt="ball"/>
+      <img src="Assets/Rectangle7.png" class="bar" alt="bar"/>
+      <img src="Assets/Ellipse11.png" class="deep_left" alt="left_ellipse"/>
+      <img src="Assets/Rectangle4.png" class="light_right" alt="right_box"/>
+      <img src="Assets/Ellipse3.png" class="deep_right" alt="right_ellispe"/>
+      <img src="Assets/Rectangle6.png" class="light_left" alt="left_box"/>
```

Comment:
- [OPTIONAL] I suggest you select the group of these images from the Figma design and include it as a background and not use the `img` tag.
For instance, looking at the Figma design snippet below, all the headline background images has been grouped under **"header-shapes mobile"**
![Capture](https://user-images.githubusercontent.com/26796167/148481739-3fd87931-9f67-4f97-8b9e-68ae911d566d.PNG)